packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -136,6 +136,8 @@ class BottomSheet extends Component {
 				onBackdropPress={ this.props.onClose }
 				onBackButtonPress={ this.props.onClose }
 				onSwipe={ this.props.onClose }
+				onDismiss={ Platform.OS === 'ios' ? this.props.onDismiss : undefined }
```

Comment:
I'm not sure I understand how `props.onDismiss` is different from `props.onClose` here, could you expand on this?

Reply:
`props.onDismiss` is used for calling functions to occur **after** the Modal is done dismissing. We use this to check if we should open another modal for Image size settings. 

`props.onClose` is assigned the function that will perform the action that closes the modal. So in image editor.native.js for example, we assign a `onImageSettingsClose` function that sets state.showSettings to false.